packages/sprotty/src/features/edge-junction/junction-postprocessor.ts

```diff
@@ -0,0 +1,40 @@
+/********************************************************************************
+ * Copyright (c) 2024 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable } from "inversify";
+import { IVNodePostprocessor } from "../../base/views/vnode-postprocessor";
+import { VNode } from "snabbdom";
+import { Action } from "sprotty-protocol";
+import { SModelElementImpl } from "../../base/model/smodel";
+
+@injectable()
+export class JunctionPostProcessor implements IVNodePostprocessor {
+    decorate(vnode: VNode, element: SModelElementImpl): VNode {
+        return vnode;
+    }
+    postUpdate(cause?: Action | undefined): void {
+        const svg = document.querySelector('svg#sprotty_root > g');
```

Comment:
Can we guarantee that the root element always has the id `sprotty_root`? Is it hard-coded somewhere in the framework?

Reply:
No, we can't. The selector should start from the `baseDiv` specified in the viewer options.